openapi.yaml

```diff
@@ -60,6 +60,45 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /validate:
+    post:
+      summary: Validate the given AsyncAPI document.
+      operationId: validate
+      tags:
+        - validate
+        - template
+      requestBody:
+        description: Validate the given AsyncAPI document with the AsyncAPI parser.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Template'
+      responses:
+        '200':
+          description: The given template is valid.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Template'
```

Comment:
It should have different responses and of course it shouldn't be a `Template` schema - see my previous comment about `/validate` path

Reply:
yup, totally agreed 👍 👍